lib/span.js

```diff
@@ -262,25 +391,25 @@ Span.prototype.runAsync = function (fn) {
   // it, then runs the user's runner function. That way our wrapper is invoked
   // before the user's callback. handler is used only for memcached. No other
   // callback function supplies it.
-  const ret = fn.call(span, (cb, handler) => ao.bind(function (err) {
+  const ret = fn.call(span, (cb, handler) => ao.bind(function (error) {
     if (handler) {
       // handler is present only for some memcached functions.
       // TODO BAM how to handle this with customTxName...
       handler.apply(this, arguments)
     } else {
       // this is the "normal", i.e., non-memcached path.
-
+      const et = (Date.now() - startTime) * 1000;
```

Comment:
funny - it moved because an intermediate approach to solving the span exit required it. i just left it there but probably shouldn't because it creates a bit of extra work for regular spans.

Reply:
addressed